tests/core/full_node/test_mempool.py

```diff
@@ -2061,6 +2061,30 @@ def generator_condition_tester(
 
 
 class TestGeneratorConditions:
+    @pytest.mark.parametrize("rust_checker", [True, False])
+    def test_invalid_condition_args_terminator(self, rust_checker: bool):
+
+        # note how the condition argument list isn't correctly terminated with a
+        # NIL atom. This is allowed, and all arguments beyond the ones we look
+        # at are ignored, including the termination of the list
```

Comment:
Could be a little more explicit, and say "The conditions checking code only reads up to the "50" below